src/com/twitter/intellij/pants/execution/PantsMakeBeforeRun.java

```diff
@@ -111,6 +112,14 @@ else if (runConfiguration instanceof CommonProgramRunConfigurationParameters) {
       if (buildRoot.isPresent()) {
         ((CommonProgramRunConfigurationParameters) runConfiguration).setWorkingDirectory(buildRoot.get().getPath());
       }
+
+      if(runConfiguration instanceof JUnitConfiguration) {
+        Optional<String[]> os = PantsOptions.getPantsOptions(project)
+          .flatMap(options -> options.getList(PantsConstants.PANTS_OPTION_TEST_JUNIT_OPTIONS));
```

Comment:
This introduces a minor lag in the UI if PantsOptions wasn't cached. We could either the following options:
1) Show the UI `Getting Pants options ...`
2) Preemptively populate pants options up on project start.

Reply:
![lag](https://user-images.githubusercontent.com/1066003/69737081-08184000-1134-11ea-81fd-63d6e8c7940d.gif)

Reply:
Looks great. Thanks!